src/validations/MeasureSchemaValidator.ts

```diff
@@ -8,9 +8,6 @@ export const MeasureSchemaValidator = Yup.object().shape({
     .required("A Measure name is required.")
     .matches(/[a-zA-Z]/, "A Measure name must contain at least one letter.")
     .matches(/^((?!_).)*$/, "Measure Name must not contain '_' (underscores)."),
-  model: Yup.string()
```

Comment:
is MeasureSchemaValidator still used anywhere?

Reply:
@nmorasb No it's not, I checked before I removed it. Thanks Nate!